### PR TITLE
feat(prerequisites): use "tdp-collection-prerequisites" repository as git submodule

### LINKED ISSUE #165: Add "prerequisites" feature by using "tdp-collection-prerequisites" repository
Use https://github.com/TOSIT-IO/tdp-collection-prerequisites as git submodule to configure tdp prerequisites if the `-e prerequisites` is defined for the `setup` script.

### PR DESCRIPTION
**Pull request by rpignolet**
#### Which issue(s) this PR fixes:

Fixes #165 

#### Additional comments:

To make clear that you license your contribution under
the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0) and that you give permission to [TOSIT](https://www.tosit.io/),
you have to acknowledge this by using the following check-box.

 - [x] I hereby declare this contribution to be licenced under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)

 - [x] I hereby agree to grant [TOSIT](https://www.tosit.io/) a copyright license to use my contributions.

### CODE REVIEW

**Review by gboutry — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -98,33 +102,64 @@ For TDP Vagrant usage see https://github.com/TOSIT-IO/tdp-vagrant.
...
+This playbook deploys the following services: Chrony, a CA.
 
-#### SSH Key Generation and Deployment
+#### HTTP proxy configuration
 
-It is **optionally** possible to generate a new ssh key pair and deploy the public key to each host, though `vagrant ssh <ansible-host>` works just fine in the context of this getting-started cluster. Use the below command to generate SSH keys and deploy them throughout the cluster:
+You can configure HTTP proxy with Ansible variables:
+- `http_proxy`
+- `https_proxy`
+- `no_proxy`
```
Comment (leopaul36) [inline README.md:120]: It's a bit unclear what these variables will do, this part needs more context IMO

Comment (leopaul36) [reply README.md:120]: I just noticed it is documented at `inventory/group_vars/all.yml`, we don't need this in the main README IMO

Comment (rpignolet) [reply README.md:120]: @leopaul36 I remove the "HTTP proxy configuration" section ?

Comment (gboutry) [reply README.md:120]: This means, people using this, can't get information about proxy from the readme ?

I think, at least, there should be a mention to proxy in the readme. (even if it's just telling you to go to the group_vars)

Comment (rpignolet) [reply README.md]: Documentation added with https://github.com/TOSIT-IO/tdp-collection-prerequisites/pull/9.

**Comment by rpignolet** [comment]
Last force push I added "Configure prerequisites" to the table of content.

**Review by leopaul36 — APPROVED** [review]

**Comment by rpignolet** [comment]
Last and final force push to update git submodule commit for tdp-collection-prerequisites.

**Pull request merged**